static/js/biomedical/disease/drug_table.tsx

```diff
@@ -1,37 +1,59 @@
 import React from "react";
 
 import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
-import { CompoundDiseaseTreatmentData } from "./chart";
-
-export interface DrugTableProps {
-  data: CompoundDiseaseTreatmentData[];
+export interface DrugTableColumnType {
+  // the column id
+  id: string;
+  // the column display name or header
+  name: string;
 }
-
-export function DrugTable(props: DrugTableProps): JSX.Element {
+export interface DrugDataType {
+  // compound node
+  node: string;
+  // compound id
+  id: string;
+  // compound name
+  name: string;
+  // clinical phase number or drug source
+  [key: string]: any;
+}
+export interface DrugTreatmentTableProps {
+  // stores the column id and column header name
```

Comment:
please also mention that this stores the order of the columns and that column id should match the keys of the objects in the data

Reply:
Done!